react/components/NumericStepper/index.js

```diff
@@ -25,6 +25,10 @@ const validateValue = (value, min, max, defaultValue) => {
 }
 
 const formattedDisplayValue = (value, unitMultiplier, suffix) => {
+  if (Number.EPSILON === undefined) {
```

Comment:
The fix should be adding the polyfill of EPSILON in our polyfill.io solution 

https://polyfill.io/v3/url-builder/

![Uploading image.png…]()